core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -32,6 +32,7 @@ private CatalogProperties() {}
   public static final String VIEW_DEFAULT_PREFIX = "view-default.";
   public static final String VIEW_OVERRIDE_PREFIX = "view-override.";
   public static final String METRICS_REPORTER_IMPL = "metrics-reporter-impl";
+  public static final String HTTP_HEADER_PREFIX = "header.";
```

Comment:
this is specific to REST, so shouldn't be in `CatalogProperties`. `RESTUtil` would be the right place but I'm currently not sure whether we should be exposing this